NLibrary/Classes/ProfileMVVM/Mocks/ProfileMockRepo.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public class ProfileMockRepo: ProfileRepoProtocol {
+    var closurePathway: APIRequestResult = .succesfullRequest
+    public required init () {
+    }
+
+    public func getUserProfile(completion: @escaping(Result<ProfileModel, APIFailureResult>) -> Void) {
+        switch closurePathway {
+        case .failedRequest(_):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Arguments can be omitted when matching enums with associated types if they are not used.](https://app.codacy.com/manual/NathanDVT/NLibrary/pullRequest?prid=5389739)